edx_event_bus_kafka/internal/consumer.py

```diff
@@ -156,8 +150,8 @@ def emit_signals_from_message(self, msg):
         # CloudEvents specifies using UTF-8 for header values, so let's be explicit.
         event_type_str = event_type.decode("utf-8")
 
-        # If we get a message with the wrong signal encoding, we do not want to send it along.
-        # TODO (EventBus): Handle this particular sad path more gracefully.
+        # TODO: Accept multiple event types: https://github.com/openedx/openedx-events/issues/78
```

Comment:
It may be useful to have that issue point to this code, but I don't think we need a TODO for this. We'll do this if and when we determine we need it.

Reply:
Yeah, can delete.